account/config.go

```diff
@@ -55,7 +55,7 @@ type UpdateConfigInput struct {
 func (c *ConfigClient) Update(ctx context.Context, input *UpdateConfigInput) (*Config, error) {
 	path := fmt.Sprintf("/%s/config", c.client.AccountName)
 	reqInputs := client.RequestInput{
-		Method: http.MethodPut,
+		Method: http.MethodPost,
```

Comment:
1) Why the change from `PUT` to `POST`?

2) Someday we should come through and replace the `fmt.Sprintf()` calls with `path.Join("/", c.client.AccountName, "config"`.  Not now, but at some point...

Reply:
So the change from PUT to POST was to follow the convention of the other Update API endpoints in this package which are all POST

Reply:
I reverted this - I am not sure this is the correct thing to do - I shouldn't change an endpoint that is already in place :)